examples/core/rgcn/hetero_rgcn.py

```diff
@@ -0,0 +1,647 @@
+"""
+This script, `hetero_rgcn.py`, trains and tests a Relational Graph
+Convolutional Network (R-GCN) model for node classification on the
+Open Graph Benchmark (OGB) dataset "ogbn-mag". For more details on
+"ogbn-mag", please refer to the OGB website:
+(https://ogb.stanford.edu/docs/linkprop/)
+
+Paper [Modeling Relational Data with Graph Convolutional Networks]
+(https://arxiv.org/abs/1703.06103).
+
```

Comment:
Worth to add a paragraph to talk about what's the difference between homograph node classification and hererograph node classification.

e.g. why do we need generate embeddings?

Reply:
Get, I'll explain the difference in embedding generation.